samples/apps/copilot-chat-app/importdocument/Program.cs

```diff
@@ -1,13 +1,10 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
[nit] If you feel inspired, take a look at https://github.com/dotnet/command-line-api/blob/main/docs/Your-first-app-with-System-CommandLine-DragonFruit.md. It would simplify the Main method's command-line argument parsing.

Reply:
I will address this in a separate PR.